alphastats/DataSet_Preprocess.py

```diff
@@ -30,45 +31,58 @@ def preprocess_print_info(self):
         print(pd.DataFrame(self.preprocessing_info.items()))
 
     def _remove_na_values(self, cut_off):
+        if (
+            self.preprocessing_info.get("Missing values were removed")
```

Comment:
I don't know much about this `preprocessing_info`, but it seems to store information in human-readable keys.
This could lead to problems (say, I access something as `self.preprocessing_info.get("Missing values were removed.")` (did you spot the trailing dot? ;-))
I would suggest introducing a set of string constants, e.g.
`MISSING_VALUES_REMOVED = "Missing values were removed"`
somewhere and access this store exclusively through them

(not now, just for the future)